.gitignore

```diff
@@ -348,3 +348,8 @@ MigrationBackup/
 
 # Ionide (cross platform F# VS Code tools) working folder
 .ionide/
+
+
+# Custom
+package.json
```

Comment:
cant ignore these files in git. Need them in the /ClientApp/** folder for when implementing libraries

Reply:
Fixed this by adding `/` in front of the filename in .gitignore. This way it only ignores package.json and package-lock.json in the root folder. https://stackoverflow.com/questions/3637660/how-to-exclude-file-only-from-root-folder-in-git

Reply:
Thats a great solution